src/generator/AutoRest.CSharp/Model/ControllerToModelBinderMapping.cs

```diff
@@ -0,0 +1,31 @@
+using AutoRest.Core.Model;
+using System.Collections.Generic;
+namespace AutoRest.CSharp.Model
+{
+
+    static class ControllerToModelBinderMapping
+    {
```

Comment:
Urg. I really dislike static dictionaries for translation purposes. It's memory wasteful, a maintenance hazard, not particularly efficient, and the compiler can't do anything to inline it. 

I'd rather go with something a bit slimmer, and simpler:
``` csharp
public static string GetModelBinder(this ParameterLocation parameterLocation )
{
  switch( parameterLocation ) {
    case ParameterLocation.None:
      return string.Empty;
    case ParameterLocation.Path:
      return "FromRoute";
    case ParameterLocation.FormData:
      return "FromForm";
    default:
      return $"From{parameterLocation}";
  }
}
```